From 6cce7f590419e74e5e43fd10be1883fae451bcad Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 19 Sep 2014 12:29:32 -0400
Subject: [acpi] Remove duplicate definitions of PREFIX

Message-id: <1411129776-21656-58-git-send-email-prarit@redhat.com>
Patchwork-id: 96387
O-Subject: [RHEL7.1 PATCH BZ 1141846 57/61] ACPI: Remove duplicate definitions of PREFIX
Bugzilla: 1141846
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Myron Stowe <mstowe@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1141846

commit 07070e12cf5cab46a783a4753f07bb99e557473b
Author: Hanjun Guo <hanjun.guo@linaro.org>
Date:   Thu Mar 13 12:47:39 2014 +0800

    ACPI: Remove duplicate definitions of PREFIX

    We already have a macro for PREFIX of "ACPI: " in
    drivers/acpi/internal.h, so remove the duplicate ones
    in ACPI drivers when internal.h is included.

    Signed-off-by: Hanjun Guo <hanjun.guo@linaro.org>
    Signed-off-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Some minor changes due to code drift so we have to also add internal.h.

Cc: Lenny Szubowicz <lszubowi@redhat.com>
Cc: Stefan Assmann <sassmann@redhat.com>
Cc: Tony Camuso <tcamuso@redhat.com>
Cc: Don Zickus <dzickus@redhat.com>
Cc: Dean Nelson <dnelson@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/acpi_cmos_rtc.c b/drivers/acpi/acpi_cmos_rtc.c
index 84190ed..961b45d 100644
--- a/drivers/acpi/acpi_cmos_rtc.c
+++ b/drivers/acpi/acpi_cmos_rtc.c
@@ -18,8 +18,6 @@
 
 #include "internal.h"
 
-#define PREFIX "ACPI: "
-
 ACPI_MODULE_NAME("cmos rtc");
 
 static const struct acpi_device_id acpi_cmos_rtc_ids[] = {
diff --git a/drivers/acpi/container.c b/drivers/acpi/container.c
index 9c35765..6c05ffd 100644
--- a/drivers/acpi/container.c
+++ b/drivers/acpi/container.c
@@ -31,8 +31,6 @@
 
 #include "internal.h"
 
-#define PREFIX "ACPI: "
-
 #define _COMPONENT   ACPI_CONTAINER_COMPONENT
 ACPI_MODULE_NAME("container");
 
diff --git a/drivers/acpi/dock.c b/drivers/acpi/dock.c
index c215a92..3d9506a 100644
--- a/drivers/acpi/dock.c
+++ b/drivers/acpi/dock.c
@@ -37,8 +37,6 @@
 
 #include "internal.h"
 
-#define PREFIX "ACPI: "
-
 #define ACPI_DOCK_DRIVER_DESCRIPTION "ACPI Dock Station Driver"
 
 ACPI_MODULE_NAME("dock");
diff --git a/drivers/acpi/osl.c b/drivers/acpi/osl.c
index 1679331..5be572a 100644
--- a/drivers/acpi/osl.c
+++ b/drivers/acpi/osl.c
@@ -57,7 +57,7 @@
 
 #define _COMPONENT  ACPI_OS_SERVICES
 ACPI_MODULE_NAME("osl");
-#define PREFIX  "ACPI: "
+
 struct acpi_os_dpc {
  acpi_osd_exec_callback function;
  void *context;
diff --git a/drivers/acpi/pci_link.c b/drivers/acpi/pci_link.c
index 2652a61..9824a4d 100644
--- a/drivers/acpi/pci_link.c
+++ b/drivers/acpi/pci_link.c
@@ -43,7 +43,7 @@
 #include <acpi/acpi_bus.h>
 #include <acpi/acpi_drivers.h>
 
-#define PREFIX "ACPI: "
+#include "internal.h"
 
 #define _COMPONENT   ACPI_PCI_COMPONENT
 ACPI_MODULE_NAME("pci_link");
diff --git a/drivers/acpi/pci_root.c b/drivers/acpi/pci_root.c
index deb4a97..3973292 100644
--- a/drivers/acpi/pci_root.c
+++ b/drivers/acpi/pci_root.c
@@ -41,8 +41,6 @@
 
 #include "internal.h"
 
-#define PREFIX "ACPI: "
-
 #define _COMPONENT  ACPI_PCI_COMPONENT
 ACPI_MODULE_NAME("pci_root");
 #define ACPI_PCI_ROOT_CLASS  "pci_bridge"
diff --git a/drivers/acpi/power.c b/drivers/acpi/power.c
index 3d59c0e..684d57e 100644
--- a/drivers/acpi/power.c
+++ b/drivers/acpi/power.c
@@ -47,8 +47,6 @@
 #include "sleep.h"
 #include "internal.h"
 
-#define PREFIX "ACPI: "
-
 #define _COMPONENT   ACPI_POWER_COMPONENT
 ACPI_MODULE_NAME("power");
 #define ACPI_POWER_CLASS  "power_resource"
diff --git a/drivers/acpi/processor_core.c b/drivers/acpi/processor_core.c
index 787c255..5a4f4f5 100644
--- a/drivers/acpi/processor_core.c
+++ b/drivers/acpi/processor_core.c
@@ -16,7 +16,6 @@
 
 #include "internal.h"
 
-#define PREFIX   "ACPI: "
 #define _COMPONENT  ACPI_PROCESSOR_COMPONENT
 ACPI_MODULE_NAME("processor_core");
 
diff --git a/drivers/acpi/processor_driver.c b/drivers/acpi/processor_driver.c
index aa12472..12d283d 100644
--- a/drivers/acpi/processor_driver.c
+++ b/drivers/acpi/processor_driver.c
@@ -41,8 +41,6 @@
 
 #include "internal.h"
 
-#define PREFIX "ACPI: "
-
 #define ACPI_PROCESSOR_NOTIFY_PERFORMANCE 0x80
 #define ACPI_PROCESSOR_NOTIFY_POWER 0x81
 #define ACPI_PROCESSOR_NOTIFY_THROTTLING 0x82
diff --git a/drivers/acpi/sysfs.c b/drivers/acpi/sysfs.c
index 76dd845..d962d0b 100644
--- a/drivers/acpi/sysfs.c
+++ b/drivers/acpi/sysfs.c
@@ -12,8 +12,6 @@
 #define _COMPONENT  ACPI_SYSTEM_COMPONENT
 ACPI_MODULE_NAME("sysfs");
 
-#define PREFIX "ACPI: "
-
 #ifdef CONFIG_ACPI_DEBUG
 /*
  * ACPI debug sysfs I/F, including:
diff --git a/drivers/acpi/video.c b/drivers/acpi/video.c
index 1cf9b03..20a784f 100644
--- a/drivers/acpi/video.c
+++ b/drivers/acpi/video.c
@@ -44,7 +44,7 @@
 #include <linux/suspend.h>
 #include <acpi/video.h>
 
-#define PREFIX "ACPI: "
+#include "internal.h"
 
 #define ACPI_VIDEO_BUS_NAME  "Video Bus"
 #define ACPI_VIDEO_DEVICE_NAME  "Video Device"
diff --git a/drivers/acpi/video_detect.c b/drivers/acpi/video_detect.c
index ddefa5f..00bcfb9 100644
--- a/drivers/acpi/video_detect.c
+++ b/drivers/acpi/video_detect.c
@@ -38,7 +38,7 @@
 #include <linux/dmi.h>
 #include <linux/pci.h>
 
-#define PREFIX "ACPI: "
+#include "internal.h"
 
 ACPI_MODULE_NAME("video");
 #define _COMPONENT  ACPI_VIDEO_COMPONENT
-- 
1.7.1